src/ccl_background.c

```diff
@@ -1023,7 +1022,10 @@ void ccl_scale_factor_of_chis(ccl_cosmology * cosmo, int nchi, double chi[], dou
 
 double ccl_growth_factor(ccl_cosmology * cosmo, double a, int * status)
 {
-  if(a>1.) {
+  if(a==1.){
```

Comment:
Okay, this is a little bit of a hack. Let me explain: as part of my coding it was necessary to 'upgrade' the function sigma(R) so that it always took the scale factor as an additional argument (which defaults to a=1 in python). This meant that I changed sigma(R) -> sigma(R,a)=g(a)*sigma(R,a=1). This means that every time the new sigma(R,a) is now accessed the growth function is also accessed. This meant that some of the tests that were previously passing for massive-neutrino cosmologies ended up failing because the neutrino cosmologies do not allow the growth function to be accessed because it is strongly scale dependent. As far as I remember these tests were of the mass function for massive-neutrino models. The only failed because they access the growth function. sigma(R,a=1) is well defined for the massive neutrino cosmologies because you have the power spectrum at a=1, so I figured the quickest way to prevent these fails was to enforce g(a=1)=1 always.

I figured it wouldn't be a problem because I couldn't imagine a scenario where you wouldn't want g(a=1)=1... but it is a bit of a hack. Happy to hear of any alternatives or better ideas to circumvent this problem.